modules/ossm-remove-operators.adoc

```diff
@@ -6,105 +6,22 @@
+* {ProductName}
+* Kiali
+* Jaeger
+* OpenShift Elasticsearch
 
 .Procedure
 
 . Log in to the {product-title} web console.
 
-. From the *Operators* → *Installed Operators* page, scroll or type a keyword into
-the *Filter by name* to find the Kiali Operator. Then, click on it.
-
-. On the right-hand side of the *Operator Details* page, select *Uninstall
-Operator* from the *Actions* drop-down menu.
-
-. When prompted by the *Remove Operator Subscription* window, optionally select the
-*Also completely remove the Operator from the selected namespace*
-check box if you want all components related to the installation to be removed.
-This removes the CSV, which in turn removes the pods, Deployments, CRDs, and CRs
-associated with the Operator.
-
-[id="ossm-remove-operator-jaeger_{context}"]
-== Removing the Jaeger Operator
-
-Follow this procedure to remove the Jaeger Operator.
-
-.Prerequisites
-
-* Access to the {product-title} web console.
-* The Jaeger Operator must be installed.
-
-.Procedure
-
-. Log in to the {product-title} web console.
-
-. From the *Operators* → *Installed Operators* page, scroll or type a keyword into
-the *Filter by name* to find the Jaeger Operator. Then, click on it.
-
-. On the right-hand side of the *Operator Details* page, select *Uninstall
-Operator* from the *Actions* drop-down menu.
-
-. When prompted by the *Remove Operator Subscription* window, optionally select the
-*Also completely remove the Operator from the selected namespace*
-check box if you want all components related to the installation to be removed.
-This removes the CSV, which in turn removes the pods, Deployments, CRDs, and CRs
-associated with the Operator.
-
-[id="ossm-remove-operator-elasticsearch_{context}"]
-== Removing the OpenShift Elasticsearch Operator
-
-Follow this procedure to remove the OpenShift Elasticsearch Operator.
-
-.Prerequisites
-
-* Access to the {product-title} web console.
-* The OpenShift Elasticsearch Operator must be installed.
-
-.Procedure
-
-. Log in to the {product-title} web console.
-
-. From the *Operators* → *Installed Operators* page, scroll or type a keyword into
-the *Filter by name* to find the OpenShift Elasticsearch Operator. Then, click on it.
-
-. On the right-hand side of the *Operator Details* page, select *Uninstall
-Operator* from the *Actions* drop-down menu.
+. From the *Operators* → *Installed Operators* page, scroll or type a keyword into the *Filter by name* to find the each Operator. Then, click on it.
 
-. When prompted by the *Remove Operator Subscription* window, optionally select the
-*Also completely remove the Operator from the selected namespace*
-check box if you want all components related to the installation to be removed.
-This removes the CSV, which in turn removes the pods, Deployments, CRDs, and CRs
-associated with the Operator.
+. On the the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu. Follow the prompts to uninstall each Operator.
```

Comment:
"drop-down menu" makes me twitch.  Please check the ISG "Usage and highlighting for interface elements" around p239 in _my copy_ and see if "*Actions* list" would match the ISG guidance for the menu.